Library/Homebrew/os/mac/version.rb

```diff
@@ -90,6 +96,9 @@ def requires_nehalem_cpu?
       alias requires_sse41? requires_nehalem_cpu?
       alias requires_sse42? requires_nehalem_cpu?
       alias requires_popcnt? requires_nehalem_cpu?
+
+      # Represents the absence of a version.
+      NULL = Version.new("10.0").tap { |v| v.instance_variable_set(:@version, nil) }.freeze
```

Comment:
Why `10.0`?

Reply:
This needs some arbitrary version, since we can't call `new` with `nil`.

Reply:
Ok, cool. Can a comment explain that? `0.0` may be a better fit given `10.0` looks a lot like a macOS version 😁

Reply:
Well it needs look like a macOS version since the constructor checks this using a regex.

Reply:
Ok, that's fine, again comment would be handy here.